input/blog/posts/net-foundation-election-2021-nominations-open.md

```diff
@@ -0,0 +1,16 @@
+---
+Title: .NET Foundation Election 2021 Nominations are Open!
+Author: Iris Classon
+Published: 2021-07-14
+---
+
+## .NET Foundation Election 2021 Nominations are Open!
+
+It’s that time of the year again dear friends- the .NET nominations! On behalf of the board and nomination committee we would like to welcome all of you to participate- be that by [applying as a candidate yourself](https://forms.office.com/Pages/ResponsePage.aspx?id=8ru98HbNZ0mMTFCoAnRpEmp3qxxG5iNBl2dHcSfTzztUOEdKT05ENERURFFIMTFOSDU3TE81NFJDQi4u), suggesting somebody else or simply preparing to vote and help spread the word.
```

Comment:
Suggested:

"It's that time of the year again, dear friends - nominations are now open for the 2021 .NET Foundation Election! On behalf of the Board and Nomination Committee, we would like to welcome all of you to participate in whatever way you can: by applying as a candidate yourself, suggesting somebody else, or simply preparing to vote and help spread the word."